### PR TITLE
Added 20 more English quotes

### PR DESCRIPTION
**Pull request by holyhearted**
this isn’t actually 36 commits codespaces is just dumb

### CODE REVIEW

**Comment by Ferotiq** [comment]
Hey it looks like you are just doing all of your changes on the `master` branch of your fork, it would be nice if for each PR you did, you made a new branch on your fork, that way your previous changes don't end up cluttering the commit history in your PRs.
Not a huge deal though.

**Comment by holyhearted** [comment]
> Hey it looks like you are just doing all of your changes on the `master` branch of your fork, it would be nice if for each PR you did, you made a new branch on your fork, that way your previous changes don't end up cluttering the commit history in your PRs.
> Not a huge deal though.

ahhhhhh! thank you very much, i’ll definitely do that in the future

File: frontend/static/quotes/english.json [hunk 1]
```diff
@@ -34998,6 +34998,126 @@
...
+      "text": "I covered you in royal jelly, I made you queen and you forgave me.",
+      "source": "The Waves Have Come",
+      "length": 66,
+      "id": 6137
+    },
+    {
+      "text": "Everyone is here, but you're nowhere near.",
+      "source": "Nowhere Near",
+      "length": 42,
+      "id": 6138
```
Comment (Miodec) [inline frontend/static/quotes/english.json:35066]: Remove or replace with a longer version

**Review by Miodec — CHANGES_REQUESTED** [review]
Most of the sources seem weird? They are just a few words from the quote, or the general idea of the quote. No authors, books, anything?

File: frontend/static/quotes/english.json [hunk 2]
```diff
@@ -34998,6 +34998,126 @@
...
+    },
+    {
+      "text": "The last time I saw Richard was Detroit in '68, and he told me, \"All romantics meet the same fate someday: cynical and drunk, and boring someone in some dark cafe.\"",
+      "source": "The Last Time I Saw Richard",
+      "length": 164,
+      "id": 6129
+    },
+    {
+      "text": "The hours pass, and she still counts the minutes that I am not there. I swear I didn't mean for it to feel like this, like every inch of me is bruised, bruised.",
+      "source": "Bruised",
```
Comment (holyhearted) [inline frontend/static/quotes/english.json:35016]: Jack’s Mannequin — Bruised

File: frontend/static/quotes/english.json [hunk 3]
```diff
@@ -34998,6 +34998,126 @@
...
+      "length": 160,
+      "id": 6130
+    },
+    {
+      "text": "Can't keep hangin' on to what is dead and gone. If you built yourself a myth, you'd know just what to give. Materialize, or let the ashes fly?",
+      "source": "Myth",
```
Comment (holyhearted) [inline frontend/static/quotes/english.json:35022]: Beach House — Myth

File: frontend/static/quotes/english.json [hunk 4]
```diff
@@ -34998,6 +34998,126 @@
...
+      "length": 142,
+      "id": 6131
+    },
+    {
+      "text": "\"More.\" You want more, you tell me \"more.\" Only time can run me.",
```
Comment (holyhearted) [inline frontend/static/quotes/english.json:35027]: Beach House — Walk In The Park

File: frontend/static/quotes/english.json [hunk 5]
```diff
@@ -34998,6 +34998,126 @@
...
+      "source": "Walk In The Park",
+      "length": 64,
+      "id": 6132
+    },
+    {
+      "text": "Yet I'm tracing figure eights on ice in skates, so well. And if this ice should break, it would be my mistake.",
+      "source": "PPP",
```
Comment (holyhearted) [inline frontend/static/quotes/english.json:35034]: Beach House — PPP

File: frontend/static/quotes/english.json [hunk 6]
```diff
@@ -34998,6 +34998,126 @@
...
+      "length": 110,
+      "id": 6133
+    },
+    {
+      "text": "Don't you ever cross that bridge in your mind again. It's like a movie screen.",
+      "source": "Movie Screen",
```
Comment (holyhearted) [inline frontend/static/quotes/english.json:35040]: Chelsea Wolfe — Movie Screen

File: frontend/static/quotes/english.json [hunk 7]
```diff
@@ -34998,6 +34998,126 @@
...
+      "length": 78,
+      "id": 6134
+    },
+    {
+      "text": "I want flatlands - soft and steady breeze bringing scents of lined-up orchard trees, dripping heavy with pears and dancing leaves. I want flatlands, will you go there with me?",
+      "source": "Flatlands",
```
Comment (holyhearted) [inline frontend/static/quotes/english.json:35046]: Chelsea Wolfe — Flatlands

File: frontend/static/quotes/english.json [hunk 8]
```diff
@@ -34998,6 +34998,126 @@
...
+    },
+    {
+      "text": "I covered you in royal jelly, I made you queen and you forgave me.",
+      "source": "The Waves Have Come",
+      "length": 66,
+      "id": 6137
+    },
+    {
+      "text": "Everyone is here, but you're nowhere near.",
+      "source": "Nowhere Near",
```
Comment (holyhearted) [inline frontend/static/quotes/english.json:35064]: Yo La Tengo — Nowhere Near

File: frontend/static/quotes/english.json [hunk 9]
```diff
@@ -34998,6 +34998,126 @@
...
+    },
+    {
+      "text": "Well, I used to have the notion that I could swim the ocean, if I knew you were waiting for me.",
+      "source": "The Whole of the Law",
+      "length": 95,
+      "id": 6139
+    },
+    {
+      "text": "If it's not supposed to be, then I'll just let it be. And out of everyone - be honest with me - who do you change for? Who's top of your world? And out of everyone, who's your type of girl?",
+      "source": "Pristine",
```
Comment (holyhearted) [inline frontend/static/quotes/english.json:35076]: Snail Mail — Pristine

File: frontend/static/quotes/english.json [hunk 10]
```diff
@@ -34998,6 +34998,126 @@
...
+      "length": 189,
+      "id": 6140
+    },
+    {
+      "text": "And sometimes when he looks at me, I know he needs you, you're all that he sees.",
+      "source": "Georgia",
```
Comment (holyhearted) [inline frontend/static/quotes/english.json:35082]: Phoebe Bridgers — Georgia

File: frontend/static/quotes/english.json [hunk 11]
```diff
@@ -34998,6 +34998,126 @@
...
+    },
+    {
+      "text": "And I have this dream where I'm screaming underwater, while my friends are waving from the shore. And I don't need you to tell me what that means, I don't believe in that stuff anymore.",
+      "source": "Funeral",
+      "length": 185,
+      "id": 6142
+    },
+    {
+      "text": "I'm on the outside looking through: you're throwing rocks around your room. And while you're bleeding on your back in the glass, I'll be glad that I made it out, and sorry that it all went down like it did.",
+      "source": "Motion Sickness",
```
Comment (holyhearted) [inline frontend/static/quotes/english.json:35094]: Phoebe Bridgers — Motion Sickness

File: frontend/static/quotes/english.json [hunk 12]
```diff
@@ -34998,6 +34998,126 @@
...
+      "length": 206,
+      "id": 6143
+    },
+    {
+      "text": "If you're a work of art, I'm standing too close. I can see the brushstrokes.",
+      "source": "ICU",
```
Comment (holyhearted) [inline frontend/static/quotes/english.json:35100]: Phoebe Bridgers — ICU

File: frontend/static/quotes/english.json [hunk 13]
```diff
@@ -34998,6 +34998,126 @@
...
+    },
+    {
+      "text": "Baby, you're a vampire. You want blood, and I promised - I'm a bad liar, with a savior complex.",
+      "source": "Savior Complex",
+      "length": 95,
+      "id": 6145
+    },
+    {
+      "text": "I buried a hatchet, it's coming up lavender. The future's unwritten, the past is a corridor: I'm at the exit, looking back through the hall; you are anonymous, I am a concrete wall.",
+      "source": "Smoke Signals",
```
Comment (holyhearted) [inline frontend/static/quotes/english.json:35112]: Phoebe Bridgers — Smoke Signals

File: frontend/static/quotes/english.json [hunk 14]
```diff
@@ -34998,6 +34998,126 @@
...
+      "length": 181,
+      "id": 6146
+    },
+    {
+      "text": "Just tell me about the boulevards. Tell me about your life in art.",
+      "source": "My Life in Art",
```
Comment (holyhearted) [inline frontend/static/quotes/english.json:35118]: Mojave 3 — My Life in Art

**Review by Miodec — APPROVED** [review]

**Pull request merged**